tssc/__init__.py

```diff
@@ -467,11 +549,39 @@
       - implementer: OpenSCAP
         config: {}
 
-      # WARNING: not yet implemented
-      create-deployment-environment: []
+      deploy:
+      - implementer: ArgoCD
+        config:
+          # argocd specific variables are set per environment above
 
-      # WARNING: not yet implemented
-      deploy: []
+          # Required
+          helm-config-repo: ''
+
+          # Optional
+          #values-yaml-directory: './cicd/Deployment/'
+
+          # Optional
+          #value-yaml-template: 'values.yaml.j2'
+
+          # Optional
+          #argocd-helm-chart-path: './'
+
+          # Required
+          git-email: ''
+
+          # Optional
+          #git-friendly-name: 'TSSC'
+
+          # Optional
+          #git-username: None
+
+          # Optional
+          #git-password: None
+
+          # Any template parameters required by values.yaml.j2 can be listed below.
+          # Note dashes will be converted to underscores to be compliant with the jinja
+          # template variable specification
+          readiness-probe-path: ''
```

Comment:
@akim309 @kharyam wouldn't this path need to be different per environment?

Reply:
@itewk Agreed - created https://projects.engineering.redhat.com/browse/NAPSSPO-901 to create environment specific values.yaml files